tests/configurations/unit/jest.config.js

```diff
@@ -16,11 +16,12 @@ module.exports = {
     ['jest-html-reporters', { multipleReportsUnitePath: './reports', pageTitle: 'unit', publicPath: './reports', filename: 'unit.html' }],
```

Comment:
you should test your code, you have scripts generating interfaces. I do not accept it